Utilities/DataLoading/EdFi.LoadTools/BulkLoadClient/LoadProcess.cs

```diff
@@ -37,14 +36,7 @@ public static async Task<int> Run(IConfiguration configuration)
 
                 var bulkLoadClientConfiguration = BulkLoadClientConfiguration.Create(configuration);
 
-                using var xsdFilesRetriever = new XsdFilesRetriever(
-                    bulkLoadClientConfiguration,
-                    new XsdMetadataInformationProvider(),
-                    new XsdMetadataFilesProvider(),
-                    new RemoteFileDownloader(),
-                    odsVersionInformation);
-
-                await xsdFilesRetriever.DownloadXsdFilesAsync();
+                await LoadXsdFiles(bulkLoadClientConfiguration, odsVersionInformation);
```

Comment:
should we be using ConfigureAwait(false)? Please confirm that is what we used everywhere else.